Sample/src/androidTest/java/com/shopify/testify/sample/ScreenshotRuleExampleTests.kt

```diff
@@ -71,6 +78,162 @@ class ScreenshotRuleExampleTests {
         }.assertSame()
     }
 
+    /**
+     * Demonstrates the alternative of loading layouts by resource name.
+     *
+     * This approach is useful for library projects where the R.id values are not constant.
+     */
+    @TestifyLayout(layoutResName = "com.shopify.testify.sample:layout/view_client_details")
+    @ScreenshotInstrumentation
+    @Test
+    fun usingLayoutResName() {
+        rule.setViewModifications { harnessRoot ->
+            rule.activity.getViewState(name = "usingLayoutResName").let {
+                harnessRoot.clientDetailsView.render(it)
+                rule.activity.title = it.name
+            }
+        }.assertSame()
+    }
+
+    /**
+     * Demonstrates some diagnostic information (test name) that's available to assist logging
+     */
+    @ScreenshotInstrumentation
+    @Test
+    fun diagnosticInfo() {
```

Comment:
Is this screenshot supposed to be blank?

Reply:
Yeah. It doesn't actually load anything to test.
I should probably move this to a non-screenshot test, now that you point this out. It's silly the way it is

Reply:
I decided to remove this test entirely. It didn't provide any value